plugins/styleguide/assets/javascripts/discourse/components/sections/organisms/01-post-list.gjs

```diff
@@ -0,0 +1,22 @@
+import PostList from "discourse/components/post-list";
+import { i18n } from "discourse-i18n";
+import StyleguideExample from "../../styleguide-example";
+
+const StyleguidePostList = <template>
+  <StyleguideExample
+    @title={{i18n "styleguide.sections.post_list.empty_example"}}
+  >
+    <PostList @posts="" @additionalItemClasses="styleguide-post-list-item" />
+  </StyleguideExample>
+
+  <StyleguideExample
+    @title={{i18n "styleguide.sections.post_list.populated_example"}}
+  >
+    <PostList
+      @posts={{@dummy.postList}}
+      @additionalItemClasses="styleguide-post-list-item"
+    />
+  </StyleguideExample>
+</template>;
+
+export default StyleguidePostList;
```

Comment:
Maybe in a followup, it would be nice to have a post with an expandable excerpt, that way we can see that working as well.